app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,105 @@
+class OrdersController < ApplicationController
+  
+  before_action :authenticate_user!
+  before_action :set_journey, only: %i(new create checkout)
+  before_action :set_order, only: %i(checkout charge)
+  before_action :instantiate_order, only: %i(create)
+  before_action :ensure_booking_is_active, only: %i(new checkout create)
+  
+  def checkout
+  end
+  
+  def new
+    @minutes = Order.calculate_minutes(@journey)
```

Comment:
minutes of what?

I suggest renaming this variable to something more descriptive

imparted_minutes, billable_minutes, something like that (don't know the answer for this case)